crates/zed/Cargo.toml

```diff
@@ -122,23 +122,23 @@ workspace = { workspace = true, features = ["test-support"] }
 
 [package.metadata.bundle-dev]
 icon = ["resources/app-icon-preview@2x.png", "resources/app-icon-preview.png"]
-identifier = "dev.zed.Zed-Dev"
+identifier = "dev.zed.ZedDev"
 name = "Zed Dev"
 osx_minimum_system_version = "10.15.7"
 osx_info_plist_exts = ["resources/info/*"]
 osx_url_schemes = ["zed"]
 
 [package.metadata.bundle-nightly]
 icon = ["resources/app-icon-nightly@2x.png", "resources/app-icon-nightly.png"]
-identifier = "dev.zed.Zed-Nightly"
+identifier = "dev.zed.ZedNightly"
 name = "Zed Nightly"
 osx_minimum_system_version = "10.15.7"
 osx_info_plist_exts = ["resources/info/*"]
 osx_url_schemes = ["zed"]
 
 [package.metadata.bundle-preview]
 icon = ["resources/app-icon-preview@2x.png", "resources/app-icon-preview.png"]
-identifier = "dev.zed.Zed-Preview"
+identifier = "dev.zed.ZedPreview"
```

Comment:
I'm a bit unsure whether we can just change the Apple bundle identifiers after we've already used them. Unsure because I don't know whether we use them on the backend or whether they are used when notarizing the app (see `script/bundle-mac`)

Reply:
I don't have the slightest idea about that. I only changed them because the comment in release_channels mentioned that they have to stay in sync.